salesforce-cdp/1.2/antora.yml

```diff
@@ -0,0 +1,17 @@
+name: salesforce-cdp-connector
+version: '1.2'
+display_version: 1.2 (Mule 4)
+title: Salesforce Data Cloud Connector
+nav:
+- modules/ROOT/nav.adoc
+asciidoc:
+  attributes:
+    page-component-desc: Enables you to access the Salesforce Data Cloud API.
+    page-connector-type: Connector
+    page-connector-level: Select
+    page-exchange-group-id: com.mulesoft.connectors
+    page-exchange-asset-id: mule4-sdc-connector
+    page-runtime-version: 4.3.0
```

Comment:
Shouldn't the minimum runtime should be 4.4.0 now? This may need to be changed in the connector code as well.